gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.html

```diff
@@ -32,5 +32,5 @@ <h5>{{ bannerSubtitle }}</h5>
     </mat-card-content>
   </mat-card>
 } @else {
-  <div>Coming soon.</div>
+  <app-api-card *ngFor="let api of apis" [title]="api.title" [version]="api.version" [content]="api.content"></app-api-card>
```

Comment:
What about using `@for` instead of `*ngFor`? Also, we can use a self-closing `<app-api-card />`.

Reply:
Done